glean-core/src/database/mod.rs

```diff
@@ -233,6 +233,14 @@ impl Database {
             writer.commit().unwrap();
         });
     }
+
+    /// Removes a metric from the storage (only Application Lifetime).
+    pub fn remove_application_lifetime_metric(&self, storage_name: &str, metric_name: &str) {
+       let final_key = format!("{}#{}", storage_name, metric_name);
```

Comment:
Maybe factor this out into its own function so it won't become out of sync with the code that *adds* to app_lifetime_data?

Reply:
Good point!